instances/dev/main.tf

```diff
@@ -30,16 +30,31 @@ variable "perm_env" {
   }
 }
 
+resource "aws_autoscaling_group" "dev_api" {
```

Comment:
One question we have: do we need to create this autoscaling group or will Terraform build it?